src/vivarium/framework/results/manager.py

```diff
@@ -198,7 +198,7 @@ def _add_resources(self, target: List[str], target_type: SourceType):
         if target_type == SourceType.COLUMN:
             self._required_columns.update(target)
         elif target_type == SourceType.VALUE:
-            self._required_values.update(self.get_value(target))
+            self._required_values.update([self.get_value(t) for t in target])
```

Comment:
The default value is None which is not iterable, so we will get an error

Reply:
I don't understand what you're saying.

Reply:
Understood.